notebooks/Autoencoders.ipynb

```diff
@@ -0,0 +1,632 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "z22BE9uhvoxO"
+   },
+   "source": [
+    "# Autoencoders in SciKeras"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hapoJed-voxP"
+   },
+   "source": [
+    "Autencoders are an approach to use nearual networks to distill data into it's most important features, thereby compressing the data. We will be following the [Keras tutorial](https://blog.keras.io/building-autoencoders-in-keras.html) on the topic, which goes much more in depth and breadth than we will here. You are highly encouraged to check out that tutorial if you want to learn about autoencoders in the general sense.\n",
+    "\n",
+    "\n",
+    "\n",
+    "<table align=\"left\"><td>\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/adriangb/scikeras/blob/master/notebooks/Basic_Usage.ipyn\">\n",
+    "    <img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>  \n",
+    "</td><td>\n",
+    "<a target=\"_blank\" href=\"https://github.com/adriangb/scikeras/blob/master/notebooks/Basic_Usage.ipynb\"><img width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a></td></table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iT-ibpi7voxQ"
+   },
+   "source": [
+    "### Table of contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ekJWKPFMvoxR"
+   },
+   "source": [
+    "* [Data](#Data)\n",
+    "* [Define Model](#Define-Keras-Model)\n",
+    "* [Training](#Training)\n",
+    "* [Explore Results](#Explore-Results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6avb3GBQDQyG"
+   },
+   "source": [
+    "Install SciKeras"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "qCcyTjVkvoxR"
+   },
+   "outputs": [],
+   "source": [
+    "!python -m pip install git+https://github.com/adriangb/scikeras.git@master"
```

Comment:
Oh yes, good catch. This is just so that I can test stuff, good catch. I'll change it here, and in a seperate PR fix it in all of the other notebooks.

Reply:
Fixed here and in #129